src/main/java/mate/jdbc/model/Manufacturer.java

```diff
@@ -7,6 +7,14 @@ public class Manufacturer {
     private String name;
     private String country;
 
+    public Manufacturer() {
+    }
```

Comment:
This empty constructor is unnecessary, as you have other constructors with more fields. You can safely remove this constructor.